src/metrics_classification.rs

```diff
@@ -267,6 +267,16 @@ where
     T: AsTargets<Elem = L> + Labels<Elem = L>,
 {
     fn confusion_matrix(&self, ground_truth: ArrayBase<S, Ix1>) -> Result<ConfusionMatrix<L>> {
+        self.confusion_matrix(&ground_truth)
+    }
+}
+
+impl<L: Label, S, T> ToConfusionMatrix<L, &ArrayBase<S, Ix1>> for T
```

Comment:
I'm curious how this does not run into conflicts with the trait definition above? I always thought that in the instance of two `for T` with `T` bounded the compiler runs into issues.

Reply:
I also just saw that all tests are commented out here, I'm not sure where this happened :thinking:

Reply:
I'm not sure tbh. I tried to generalize by using `std::borrow::Borrow` but it wasn't straightforward, so I just split the two implementations. Maybe I should try using `AsRef`? 

> I also just saw that all tests are commented out here, I'm not sure where this happened 🤔

I think I always saw them commented, I'll try to run them and see how it goes